src/index.js

```diff
@@ -68,11 +68,13 @@
     }
 
     //
+    this.alignOuput = opts.alignOuput === true ? true : false
```

Comment:
That's strange because `this.markdown = opts.markdown === undefined ? true : opts.markdown` seems to work.